components/docs-chef-io/content/automate/servicenow_integration_install.md

```diff
@@ -24,14 +22,15 @@ The integration between a Chef Automate server and a ServiceNow instance require
 * Chef Automate
 * Chef Automate Incident Creation Application in ServiceNow
 
-The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store. [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance. Once configured, the Chef Automate server sends HTTPS JSON notifications to the Chef Automate Incident Creation Application in a ServiceNow instance to create or update incidents.
+The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store. [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.
+Once configured, the Chef Automate server sends HTTPS JSON notifications to the Chef Automate Incident Creation Application in a ServiceNow instance to create or update incidents.
```

Comment:
```suggestion
That sends HTTPS JSON notifications from your Chef Automate installation to the ServiceNow Chef Automate Incident Creation Application to create or update incidents.
```
Add to the preceding paragraph & edit how they come together.

Reply:
@kagarmoe Do you want to add or replace line 27? Also what does "that" refer to in your suggested sentence?

Reply:
I'm leaving that to you with `& edit how they come together`.